src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/SourceInputStreamFactory.java

```diff
@@ -85,19 +90,22 @@
     private final ResetFW resetRO = new ResetFW();
 
     private final Http2PrefaceFW prefaceRO = new Http2PrefaceFW();
-    private final Http2FrameFW http2RO = new Http2FrameFW();
+    /* private */ final Http2FrameFW http2RO = new Http2FrameFW();
```

Comment:
We've found that such invasive unit tests are often too brittle to hold constant when the protocol implementation evolves, so those unit tests tend to have limited value.

Our testing strategy for all protocol correctness checks is therefore to define an appropriate specification scenario (with K3PO scripts) and then verify that the implementation meets those requirements.

We reasonably expect that the ITs will remain constant even if the protocol implementation evolves because the ITs describe the external contract made by the nukleus.

Have you already described the unit test scenarios as ITs ?